model/gamut_compress.ctl

```diff
@@ -2,17 +2,25 @@
 // <ACESuserName>ACES 1.3 Look - Gamut Compress</ACESuserName>
 
 //
-// "LMT" for compressing out of gamut scene-referred values into AP1.
-// 
-// Input and output are ACES2065-1.
+// Gamut compressor to bring out of gamut scene-referred values into AP1
 //
 
+//
+// Usage:
+//  The intended use of this transform is immediately after the IDT, so that all grading
```

Comment:
Should we say instead something along those lines:

> This transform is intended to be applied on the values produced by the conversion of camera data to AP0 by the IDT,

Saying "intended use of this transform" and then that something should occur after the IDT, does not really explain what the usage is! I don't know if what I'm saying is clear.

Reply:
I think simply "on AP0 data". No need to mention cameras, particularly as it is not certain that AP0 data originates from a camera (we specifically decided we assume nothing about the source).

Reply:
Good point for the camera, most of the IDTs are for cameras but it could be a scanner or alike.